go.mod

```diff
@@ -4,7 +4,7 @@ go 1.17
 
 require (
 	github.com/Masterminds/semver/v3 v3.1.1
-	github.com/carolynvs/magex v0.7.2
+	github.com/carolynvs/magex v0.8.0
```

Comment:
this new version gives us the ability to install binaries into a local directory instead of GOPATH/bin